test/test_nn.py

```diff
@@ -13300,6 +13300,25 @@ def test_MarginLoss_empty(self, device, dtype):
                 y = torch.ones(10, 0, device=device).type(torch.long)
                 mod(x, y)
 
+    @onlyOnCPUAndCUDA
+    def test_FractionalMaxPool2d_zero_batch(self, device):
+        mod = nn.FractionalMaxPool2d(3, output_ratio=(0.5, 0.5)).to(device)
```

Comment:
nit: No need to call `to(device)` on the module since it has no parameters or buffers.